src/main/kotlin/org/rust/lang/core/completion/lint/RsLintCompletionProvider.kt

```diff
@@ -80,7 +80,7 @@ abstract class RsLintCompletionProvider : RsCompletionProvider() {
 
     protected fun getPathPrefix(path: RsPath): String {
         val qualifier = path.qualifier ?: return ""
-        return "${getPathPrefix(qualifier)}${qualifier.referenceName}::"
+        return "${getPathPrefix(qualifier)}${qualifier.referenceName.orEmpty()}::"
```

Comment:
Looks like it produces undesired completion for `#[allow(::/*caret*/)]`

![2020-10-28 10 49 21](https://user-images.githubusercontent.com/2539310/97407161-55d09600-190b-11eb-9bc9-587552ef6414.gif)

Reply:
Not related to this PR